chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -237,7 +241,7 @@ out:
 					b.spendNotifications[op] = make(map[uint64]*spendNotification)
 				}
 				b.spendNotifications[op][msg.spendID] = msg
-				b.chainConn.NotifySpent([]*wire.OutPoint{&op})
```

Comment:
Looks like this was a duplicate call even? We already do this at notification registration site.